istio/istio/base/istio-rules.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
The convention is to name this file as `destination-rules.yaml`.

Reply:
Done